### PR TITLE
chore(STONEINTG-937): add nightly job to clean GitLab webhooks

### PR DESCRIPTION
**Pull request by dheerajodha**

### CODE REVIEW

**Comment by dheerajodha** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

**Comment by dheerajodha** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

**Comment by dheerajodha** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

**Review by dirgim — APPROVED** [review]
/lgtm if it helps :/

**Review by kasemAlem — APPROVED** [review]

**Comment by psturc** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

**Comment by dheerajodha** [comment]
/pj-rehearse periodic-ci-konflux-ci-e2e-tests-main-appstudio-dump-external-resources

File: ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh
Comment (psturc) [inline ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh:50]: this needs to be reverted before merge

Comment (dheerajodha) [reply ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh]: Reverted

**Pull request merged**
